src/core_modules/capture-core/hooks/useScopeTitleText.js

```diff
@@ -7,10 +7,15 @@ export const useScopeTitleText = (scopeId: ?string) => {
     const { trackedEntityName, programName, scopeType } = useScopeInfo(scopeId);
 
     const text = {
-        [scopeTypes.EVENT_PROGRAM]: `${programName}`,
+        [scopeTypes.EVENT_PROGRAM]: programName,
         [scopeTypes.TRACKER_PROGRAM]:
-            `${i18n.t('{{trackedEntityName}} in program', { trackedEntityName })}: ${programName}`,
-        [scopeTypes.TRACKED_ENTITY_TYPE]: `${trackedEntityName}`,
+            i18n.t('{{trackedEntityName}} in program{{escape}} {{programName}}', {
+                trackedEntityName,
+                escape: ':',
+                programName,
+                interpolation: { escapeValue: false },
```

Comment:
I'm curious: why did you find it necessary to add the escapeValue here? Tested a bit and to me it seems to work without it?

Reply:
Hey, 

I skiped the `: ` character to get rid of an error generated by the pre-commit hook.
 
<img width="1020" alt="Screenshot 2021-04-29 at 14 31 51" src="https://user-images.githubusercontent.com/6817073/116692667-0738ad00-a9bd-11eb-93d1-125f98352b2a.png">

Do you get the same error on your computer?

Reply:
Another problem I found was in the Cypress tests. The `Scenario: Navigating to the enrollment page for the same tet by clicking the link button` from the `EnrollmentPageNavigation.feature ` fails because the `programName` is not correctly translated. If I use` { escapeValue: false } `the error is corrected and the text is properly displayed. 

<img width="2212" alt="Screenshot 2021-04-30 at 14 20 58" src="https://user-images.githubusercontent.com/6817073/116694301-67305300-a9bf-11eb-8d8a-ae4fc9230197.png">

Let me know if this explains the reasons behind using it. 
Thank you!

Reply:
Interesting, you're right in the sense that I do get an error when adding the colon directly to the string. But I don't think I need the `interpolation: { escapeValue: false }` part?

Reply:
Hey and how about the Cypress failure from `Scenario: Navigating to the enrollment page for the same tet by clicking the link button` does it fail for you if you don't add `interpolation: { escapeValue: false }`? 
Thank you!

Reply:
Sorry, didn't refresh to see your last comment before replying. I give in, you're right, we do actually need this 🙂 Never encountered this problem before, good to know.

Thanks for explaining @simonadomnisoru 👍